src/main/java/duke/Task.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getDescription() {
+        return "";
+    }
+    public String getStatusIcon(boolean isDone) {
+        return (isDone ? "[\u2713] " : "[ ] "); //return tick or X symbols
```

Comment:
The comment placement seems to break off from the logical structure of the code - ideally place comments on the line before above the part you want to comment about.  

The comment also seems outdated since the function isn't returning 'X' symbol, but a blank space. So perhaps consider removing it altogether?

https://se-education.org/guides/conventions/java/basic.html#comments